lib/turnout/redis_client.rb

```diff
@@ -0,0 +1,28 @@
+begin
+  require 'redis'
+rescue LoadError
+end
+
+module Turnout
+  class RedisClient
+    @@client = nil
+
+    def self.client
+      return @@client if @@client
+      if Gem.loaded_specs.has_key?('redis') && Turnout.config.redis_url != nil
```

Comment:
This line is repeated in a few places. It would be nice to DRY it up in a method like `Turnout::RedisClient.configured?` or something like that.